components/runtime/openjdk-11/Makefile

```diff
@@ -0,0 +1,146 @@
+CONFIGURE_OPTIONS+=        --with-extra-cxxflags="$(CXXFLAGS)"
+CONFIGURE_OPTIONS+=        --with-extra-ldflags="-m64"
+CONFIGURE_OPTIONS+=        --with-version-build=$(OPENJDK_BUILD)
+CONFIGURE_OPTIONS+=        --with-cups=$(USRDIR)
+CONFIGURE_OPTIONS+=        --with-fontconfig=$(USRDIR)
+CONFIGURE_OPTIONS+=        --enable-unlimited-crypto
+CONFIGURE_OPTIONS+=        --disable-dtrace
+CONFIGURE_OPTIONS+=        --disable-hotspot-gtest
+CONFIGURE_OPTIONS+=        --disable-warnings-as-errors
+CONFIGURE_OPTIONS+=        --disable-precompiled-headers
+CONFIGURE_OPTIONS+=        --disable-ccache
+CONFIGURE_OPTIONS+=        --disable-option-checking
+
+COMPONENT_POST_BUILD_ACTION=	($(LN) -s server/libjvm.so $(@D)/build/images/jdk/lib/libjvm.so)
+
+COMPONENT_BUILD_TARGETS = images
+
+CACERTS_DIR=$(WS_TOP)/components/openindiana/ca-certificates
+CACERTS_INSTALLED=$(CACERTS_DIR)/build/$(MACH32)/.installed
+CACERTS_PROTO_DIR=$(CACERTS_DIR)/build/prototype/$(MACH)
+
+$(CACERTS_INSTALLED):
+	gmake -C $(CACERTS_DIR) install
+
+$(CACERTS_STORE): $(BUILD_DIR_64)/.built $(CACERTS_INSTALLED)
+	$(RM) $@
+	for cert in $(CACERTS_PROTO_DIR)/etc/certs/CA/*.pem; do              \
+                LC_ALL=en_US.UTF-8  $(BUILD_DIR_64)/build/images/jdk/bin/keytool \
+                  -noprompt					                                   \
+                  -importcert                                          \
+                  -keystore $@                                         \
+                  -alias `echo $$cert | $(GSED) -e 's,.*/\\([^/]*\\)\\.pem,\\1,'` \
+                  -file $$cert                                         \
+                  -storepass changeit;                                 \
+        done
+
+build:		$(BUILD_64)
+
+install: $(BUILD_DIR_64)/.built	$(CACERTS_STORE)
+	$(RM) -r $(PROTO_DIR)/usr/jdk/instances/$(OPENJDK_INSTANCE)
+	$(MKDIR) $(PROTO_DIR)/usr/jdk/instances/$(OPENJDK_INSTANCE)
+	$(CP) -a $(BUILD_DIR_64)/build/images/jdk/* $(PROTO_DIR)/usr/jdk/instances/$(OPENJDK_INSTANCE)
+	$(CP) $(CACERTS_STORE) $(PROTO_DIR)/usr/jdk/instances/$(OPENJDK_INSTANCE)/lib/security/cacerts
+	$(TOUCH) $@
+
+test:		$(NO_TESTS)
+
+# Auto-generated dependencies
+REQUIRED_PACKAGES += $(GCC_RUNTIME_PKG)
+REQUIRED_PACKAGES += $(GXX_RUNTIME_PKG)
+REQUIRED_PACKAGES += image/library/libjpeg8-turbo
+REQUIRED_PACKAGES += image/library/libpng16
+REQUIRED_PACKAGES += library/c++/harfbuzz
+REQUIRED_PACKAGES += library/giflib
+REQUIRED_PACKAGES += library/lcms2
+REQUIRED_PACKAGES += library/zlib
+REQUIRED_PACKAGES += system/library
+REQUIRED_PACKAGES += system/library/freetype-2
+REQUIRED_PACKAGES += system/library/math
+REQUIRED_PACKAGES += x11/library/libx11
+REQUIRED_PACKAGES += x11/library/libxext
+REQUIRED_PACKAGES += x11/library/libxi
+REQUIRED_PACKAGES += x11/library/libxrender
+REQUIRED_PACKAGES += x11/library/libxtst
+REQUIRED_PACKAGES += system/library/c++/sunpro
+REQUIRED_PACKAGES += system/header/header-audio
+
```

Comment:
If I run gmake REQUIRED_PACKAGES here, the last two entries aren't detected. Is library/c++/sunpro really needed?